Sources/StreamChat/Errors/ErrorPayload.swift

```diff
@@ -28,11 +28,21 @@ public struct ErrorPayload: LocalizedError, Codable, CustomDebugStringConvertibl
     }
 }
 
+/// https://getstream.io/chat/docs/ios-swift/api_errors_response/
+private enum StreamCode {
```

Comment:
Isn't it better if we call it `StreamErrorCode`? Only `StreamCode` doesn't imply that we're talking about error codes.